### PR TITLE
test: regression test for three examples

### PR DESCRIPTION
**Pull request by bwanglzu**
part of #2193 :

1. Move `multimodal` regression test into `system` folder.
2. Clean up `multimodal` regression test.
3. add `fashion` regression test similar as `multimodal`.

Reason for creating a new `system` folder:

1. regression test is another level of test above integration.
2. > can u make sure the helloworld examples tests are moved to their own folder independent of integration tests? Like this we can easily skip it when necessary much easier cc @JoanFM 

`chatbot` example is breaking, will try to fix it & add test in another PR.

### CODE REVIEW

File: tests/system/fashion/test_fashion.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+@pytest.fixture
+def query_document(test_query_image):
+    return Document(
+        {
+            'weight': 0.5,
+            'uri': test_query_image,
+            'granularity': 1,
```
Comment (JoanFM) [inline tests/system/fashion/test_fashion.py:31]: granularity is 0 in this case

File: tests/system/chatbot/test_chatbot.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        ['--workdir', str(tmpdir), '--unblock-query-flow']
+    )
+
+
+@pytest.fixture
+def query_document():
+    return Document(
+        {
+            'text': 'Is my dog safe from virus',
+            'granularity': 1,
```
Comment (JoanFM) [inline tests/system/chatbot/test_chatbot.py:25]: same here

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/system/fashion/test_fashion.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+from tests import validate_callback
+
+
+@pytest.fixture
+def helloworld_args(tmpdir):
+    return set_hw_parser().parse_args(['--workdir', str(tmpdir)])
+
+
+@pytest.fixture
+def test_query_image():
```
Comment (JoanFM) [inline tests/system/fashion/test_fashion.py:20]: this is not used right?

File: tests/system/fashion/test_fashion.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+
+@pytest.fixture
+def query_document(test_query_image):
+    return Document(content=np.random.rand(28, 28))
+
+
+def test_multimodal(helloworld_args, query_document, mocker):
```
Comment (JoanFM) [inline tests/system/fashion/test_fashion.py:29]: the name of the test is not correct

File: tests/system/fashion/test_fashion.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    """Regression test for multimodal example."""
```
Comment (JoanFM) [inline tests/system/fashion/test_fashion.py:30]: wrong annotation

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by Yongxuanzhang — COMMENTED** [review]
Very good test! Learned a lot! 😄

**Comment by bwanglzu** [comment]
@JoanFM fixed

File: tests/system/fashion/test_fashion.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+
+@pytest.fixture
+def query_document(test_query_image):
```
Comment (rutujasurve94) [inline tests/system/fashion/test_fashion.py:25]: `text_query_image` argument is not being used here

**Pull request merged**